holoviews/plotting/bokeh/__init__.py

```diff
@@ -351,6 +353,16 @@ def colormap_generator(palette):
 options.NdOverlay = Options('style', click_policy='mute')
 options.Curve = Options('style', muted_alpha=0.2)
 options.Path = Options('style', muted_alpha=0.2)
+options.Dendrogram = Options('style', muted_alpha=0.2, line_color="black")
+options.Dendrogram = Options('plot',
+    xaxis=None,
+    yaxis=None,
+    show_grid=False,
+    show_title=False,
+    show_frame=False,
+    border=0,
+    default_tools=[],
```

Comment:
I am not sure if a blank plot is the best default.

Reply:
probably fine for now, and easy enough to restore